### ISSUE TITLE
Finetune without resetting learning rate

### ISSUE THREAD

**Issue by i55code** [facebookresearch/fairseq#5250]
Finetune-from-model rests learning rate, is there a way to finetune without resetting it?
